src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
```

Comment:
From the requirements those blocks should have **additional** class stars--number. As those are the same types of blocks, i suggest `class="stars stars--0"`. Thanks to this you won't have to list all stars-number classes for common styles for those containers.